src/python_testing/TC_WEBRTCR_2_4.py

```diff
@@ -101,6 +109,18 @@ def steps_TC_WebRTCRequestor_2_4(self) -> list[TestStep]:
     def default_timeout(self) -> int:
         return 3 * 60
 
+    async def send_command(self, command):
+        async with websockets.connect(SERVER_URI) as websocket:
```

Comment:
I'm not familiar with websockets and its API. Do we need to connect for each send_command? If so is there a disconnect somewhere?